plugins/inputs/ah_wireless/ah_wireless.go

```diff
@@ -82,18 +87,18 @@ func getHDDStat(fd int, ifname string) *ah_ieee80211_hdd_stats {
 
 
      //   request.data.length = uint16(unsafe.Sizeof(cfgss));
-	request.data.length = VAP_BUFF_SIZE
+		request.data.length = VAP_BUFF_SIZE
 
         copy(request.ifrn_name[:], ah_ifname_radio2vap(ifname))
 
-	offsetsMutex.Lock()
+		offsetsMutex.Lock()
 
         if err := ah_ioctl(uintptr(fd), IEEE80211_IOCTL_GENERIC_PARAM, uintptr(unsafe.Pointer(&request))); err != nil {
-                log.Printf("Open ioctl data error %s",err)
+                log.Printf("getHDDStat ioctl data error %s",err)
```

Comment:
Consider using log.Errorf(), log.Warnf(), or log.Debugf() vs log.Printf().

Reply:
I will cleanup and re-organize logging in next commit